VideoWeb/VideoWeb.EventHub/Hub/EventHubClient.cs

```diff
@@ -127,6 +127,11 @@ private bool IsVhOfficerAsync()
             return Context.User.IsInRole(Role.VideoHearingsOfficer.DescriptionAttr());
         }
 
+        private bool IsJudgeAsync()
```

Comment:
No neeed to extend the model here, not a suitable solution for when participant IM is enabled.

Reply:
done